pkg/secrets-store/provider_client_test.go

```diff
@@ -104,6 +104,33 @@ func TestMountContent(t *testing.T) {
 				"bar": 0777,
```

Comment:
on windows:
```
            -   "bar": s"-rwxrwxrwx",
            +   "bar": s"-rw-rw-rw-",
            -   "foo": s"-rw-r--r--",
            +   "foo": s"-rw-rw-rw-",
```

Reply:
I believe the file permissions diffs are due to:

```
// On Windows, only the 0200 bit (owner writable) of mode is used; it
// controls whether the file's read-only attribute is set or cleared.
// The other bits are currently unused. For compatibility with Go 1.12
// and earlier, use a non-zero mode. Use mode 0400 for a read-only
// file and 0600 for a readable+writable file.
```

https://golang.org/src/os/file.go?s=15847:15891#L522